src/exoticatechnologies/hullmods/ExoticaTechHM.java

```diff
@@ -67,6 +68,8 @@ public static void removeFromFleetMember(FleetMemberAPI member) {
         if (shipVariant.hasHullMod("exoticatech")) {
             shipVariant.removePermaMod("exoticatech");
         }
+
+        member.updateStats();
```

Comment:
verify if this is necessary. i think it won't hurt but verify and validate.

Reply:
needs to be verified and then i'll see how it behaves without this. but first things first - make sure the "address CR comments" and the merge of #38 didn't break in the meantime so i'm working off a solid foundation.

Reply:
somehow it doesn't quite make sense to remove it, but sure, lets give it a shot.

what **does** make sense is to updateStats() when it's added and when it's removed, which is why we're updating them here in the first place. I think.

Reply:
turns out, it's not necessary at all. lets remove it.

Reply:
done. resolving.